### DISCUSSION TITLE
How to set up Stylelint in Helix?

### DISCUSSION THREAD

**Discussion by alex-kim-dev** [helix-editor/helix#8813] [Q&A]
Which language server to use, how to configure it & `languages.toml`?

**Comment by David-Else** [comment]
Maybe EFM language server with something like these settings: https://github.com/tsuyoshicho/vim-efm-langserver-settings/blob/master/config/efm-langserver/config.yaml#L396

You might be stuck as it looks like it needs the file name sending to it which needs https://github.com/helix-editor/helix/pull/6979, but I have not played about with it, hopefully someone else will chime in.

**Reply by alex-kim-dev** [reply]
Thanks for your answer, David. As I understand this will only enable formatting, but not diagnostics, code actions, etc. It would be great if it worked like in vscode with stylelint extension.

**Reply by vanarok** [reply]
The diagnostics work, but not the actions.
